src/components/Slider.vue

```diff
@@ -1,71 +1,59 @@
 <template>
-    <div class="slidecontainer">
+    <div class="nvw-slider">
         <input
             ref="slider"
-            class="slider"
+            class="nvw-slider__scroll"
             type="range"
             :min="minValue"
             :max="maxValue"
             :value="value"
-            @input="updateValue()"
-            @change="onValueChange($event)"
+            @change="valueChange ? valueChange($event, value) : null"
```

Comment:
$event should always be the last argument

Reply:
Can you review again?

Reply:
@Ulubayam it looks exactly the same to me. we have discussed this with @ozercevikaslan couple of hours back. Please use the same logic as in textfield for this. He will show you. Thanks